pennylane/ops/qubit.py

```diff
@@ -1822,6 +1822,66 @@ def adjoint(self):
         return IsingXX(-phi, wires=self.wires)
 
 
+class IsingYY(Operation):
+    r"""IsingYY(phi, wires)
+    The Ising YY coupling gate
+
+    .. math:: \mathtt{YY}(\phi) = \begin{bmatrix}
+        \cos(\phi / 2) & 0 & 0 & i \sin(\phi / 2) \\
+        0 & \cos(\phi / 2) & -i \sin(\phi / 2) & 0 \\
+        0 & -i \sin(\phi / 2) & \cos(\phi / 2) & 0 \\
+        i \sin(\phi / 2) & 0 & 0 & \cos(\phi / 2)
+        \end{bmatrix}.
+
+    **Details:**
+
+    * Number of wires: 2
+    * Number of parameters: 1
+    * Gradient recipe:
+
+    .. math::
+         \frac{d}{d\phi} f(YY(\phi))
+         = \frac{1}{2} \left[ f(YY(\phi + \pi/2)) - f(YY(\phi - \pi/2)) \right]
+
+       where :math:`f` is an expectation value depending on :math:`YY(\phi)`.
```

Comment:
Looks like the previous suggestion was incorrect. Maybe the best way is to have the gradient recipe definition be formatted just like the one for IsingXX (i.e., in two lines).